bug: Error when manually deploying

```text
 [ERROR] Missing entry-point to Worker script or to assets directory


  If there is code to deploy, you can either:
  - Specify an entry-point to your Worker script via the command line (ex: `npx wrangler deploy
  src/index.ts`)
  - Or add the following to your "wrangler.toml" file:

  ```
  main = "src/index.ts"

  ```


  If are uploading a directory of assets, you can either:
  - Specify the path to the directory of assets via the command line: (ex: `npx wrangler deploy
  --assets=./dist`)
  - Or add the following to your "wrangler.toml" file:

  ```
  [assets]
  directory = "./dist"

  ```



🪵  Logs were written to "C:\Users\photo\AppData\Roaming\xdg.config\.wrangler\logs\wrangler-2026-02-28_14-33-42_883.log"
Assertion failed: !(handle->flags & UV_HANDLE_CLOSING), file src\win\async.c, line 76
node:internal/errors:998
  const err = new Error(message);
              ^

Error: Command failed: npx wrangler deploy --config .wrangler.deploy.toml
    at genericNodeError (node:internal/errors:998:15)
    at wrappedFn (node:internal/errors:543:14)
    at checkExecSyncError (node:child_process:925:11)
    at execSync (node:child_process:997:15)
    at file:///C:/Users/photo/source/repos/CF-Architect/scripts/deploy.mjs:71:3
    at ModuleJob.run (node:internal/modules/esm/module_job:413:25)
    at async onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:654:26)
    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:101:5) {
  status: 3221226505,
  signal: null,
  output: [ null, null, null ],
  pid: 18348,
  stdout: null,
  stderr: null
}

Node.js v25.2.1
```

--- a/astro.config.mjs
+++ b/astro.config.mjs
@@ -17,6 +17,11 @@ export default defineConfig({
     build: {
       chunkSizeWarningLimit: 1500,
     },
+    resolve: {
+      alias: import.meta.env.PROD && {
+        "react-dom/server": "react-dom/server.edge",
+      },
+    },
     ssr: {
       external: ["node:crypto"],
     },

--- a/wrangler.toml
+++ b/wrangler.toml
@@ -1,5 +1,6 @@
 name = "cf-architect"
-compatibility_date = "2025-02-25"
+main = "dist/_worker.js/index.js"
+compatibility_date = "2025-02-28"
 compatibility_flags = ["nodejs_compat"]
 
 [observability]

